### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 26

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#352] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325904-85151b4d-bc60-43bb-84dc-ea4c5a0fc086.png)

**Comment by ForzaNapolixsempre** [comment]
Ponendo p=q=r=s=0 la formula risulta vera.
Quindi appartiene a SAT.
